prometheus.md

```diff
@@ -0,0 +1,195 @@
+  
+
+  
+
+## Giới thiệu cơ bản về Prometheus
+
+### Monitoring là gì? Khác gì với Logging và Tracing
+
++ `Monitoring`: là quá trình thu thập, phân tích và sử dụng thông tin có hệ thống để theo dõi tiến trình của chương trình nhằm đưa ra hướng dẫn, quyết định quản lý. Monitoring được tiến hành sau khi một chương trình đã bắt đầu và tiếp tục trong suốt thời gian thực hiện chương trình.
+
+Các monitoring tool giúp đạt được các mục tiêu này bằng cách giám sát các số liệu như liệu ứng dụng hoặc dịch vụ có phản hồi hay không, tốc độ phản hồi và mức độ sử dụng của bộ nhớ, băng thông mạng hoặc thời gian CPU.
+
++ `Logging`: được tạo bởi các ứng dụng, máy chủ, cơ sở hạ tầng mạng. Logging được sử dụng để thể hiện các biến đổi trạng thái trong một ứng dụng. Khi xảy ra sự cố, chúng tôi cần log để thiết lập thay đổi trạng thái gây ra lỗi. Hay nói cách khác, logging gi lại toàn bộ hoạt động diễn ra trong từng thời điểm. Điều này đẫn đến khó khăn trong việc lưu trữ log data.
```

Comment:
typo

Reply:
+ `Monitoring`: là quá trình thu thập, phân tích và sử dụng thông tin có hệ thống để theo dõi tiến trình của chương trình nhằm đưa ra hướng dẫn, quyết định quản lý. Monitoring được tiến hành sau khi một chương trình đã bắt đầu và tiếp tục trong suốt thời gian thực hiện chương trình.

Các monitoring tool giúp đạt được các mục tiêu này bằng cách giám sát các số liệu như: liệu ứng dụng hoặc dịch vụ có phản hồi hay không, tốc độ phản hồi và mức độ sử dụng của bộ nhớ, băng thông mạng hoặc thời gian CPU.

+ `Logging`: được tạo bởi các ứng dụng, máy chủ, cơ sở hạ tầng mạng. Logging được sử dụng để thể hiện các biến đổi trạng thái trong một ứng dụng. Khi xảy ra sự cố, chúng tôi cần log để thiết lập thay đổi trạng thái gây ra lỗi. Hay nói cách khác, logging ghi lại toàn bộ hoạt động diễn ra trong từng thời điểm. Điều này đẫn đến khó khăn trong việc lưu trữ log data.